app.rb

```diff
@@ -34,14 +34,35 @@ def list_books
     text
   end
 
-  def create_person
+  def create_teacher
     print 'Enter the person name :'
     name = gets.chomp
     print 'Enter the person age :'
     age = gets.chomp
+    print 'Enter the teacher specialization :'
+    specialization = gets.chomp
 
-    @people << Person.new(age, name)
-    puts 'Person Created Successfully'
+    @people << Teacher.new(age, specialization, name)
+    puts 'Teacher Created Successfully'
+  end
+
+  def create_student
+    print 'Enter the person name :'
+    name = gets.chomp
+    print 'Enter the person age :'
+    age = gets.chomp
+    print 'Enter classroom name :'
+    classroom = gets.chomp
+    print 'Do you have parent permission? (yes/no) :'
+    parent_permission = gets.chomp
+    parent_permission = if parent_permission == 'yes'
+                          return true
+                        else
+                          return false
+                        end
+
```

Comment:
- [ ] On running the app, I get this error
![Capture](https://user-images.githubusercontent.com/85095702/187427994-63f74507-2313-4b1c-9491-ca0be1a640e4.PNG)
Kindly remove the `return keyword` in the if statement to fix this